code/game/objects/structures/mannequin.dm

```diff
@@ -43,11 +43,16 @@
 	)
 	/// Assoc list of all item slots (turned to strings) to the items they hold.
 	var/list/worn_items = list()
+	var/list/obj/item/clothing/starting_items = list()
 
 /obj/structure/mannequin/Initialize(mapload)
 	. = ..()
 	for(var/slot_flag in slot_flags)
 		worn_items["[slot_flag]"] = null
+		for(var/obj/item/clothing/items as anything in starting_items)
+			items = new items(src)
+			if(items.slot_flags & slot_flag)
+				worn_items["[slot_flag]"] = items
```

Comment:
you wanna slap a continue here so the item doesnt get put in 2 slots at once
or make starting items an assoc list as well